engine/lite/kafka/runtime/src/main/scala/pl/touk/nussknacker/engine/lite/kafka/NuKafkaRuntimeApp.scala

```diff
@@ -26,16 +26,24 @@ object NuKafkaRuntimeApp extends App with LazyLogging {
 
   val scenario = parseScenario
 
-  val scenarioInterpreter: KafkaTransactionalScenarioInterpreter = prepareScenarioInterpreter
+  val runtimeConfig = ConfigFactory.load(ConfigFactoryExt.parseUnresolved(classLoader = getClass.getClassLoader))
 
+  val system = ActorSystem("nu-kafka-runtime", runtimeConfig)
+  import system.dispatcher
+
+  val scenarioInterpreter = prepareScenarioInterpreter(runtimeConfig)
   Runtime.getRuntime.addShutdownHook(new Thread() {
     override def run(): Unit = {
       logger.info("Closing KafkaTransactionalScenarioInterpreter")
       scenarioInterpreter.close()
     }
   })
 
-  Await.result(scenarioInterpreter.run(), Duration.Inf)
+  Await.result(for {
+    _ <- new HealthCheckServerRunner(system, scenarioInterpreter).start()
```

Comment:
We don't need to close it somehow? if e.g. exception happens in run?

Reply:
Good point, added